f600233d1b154d507b782f84604bcb12/update/sys_script_6481d45d976611106fa8b4b3f153af7f.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sys_script">
+    <sys_script action="INSERT_OR_UPDATE">
+        <abort_action>false</abort_action>
+        <access>package_private</access>
+        <action_delete>false</action_delete>
+        <action_insert>false</action_insert>
+        <action_query>false</action_query>
+        <action_update>true</action_update>
+        <active>true</active>
+        <add_message>false</add_message>
+        <advanced>true</advanced>
+        <change_fields>false</change_fields>
+        <client_callable>false</client_callable>
+        <collection>x_g_dis_atat_portfolio</collection>
+        <condition>current.name.changes() || current.portfolio_managers.changes() || current.portfolio_viewers.changes() || current.operators.changes()</condition>
+        <description/>
+        <execute_function>false</execute_function>
+        <filter_condition/>
+        <is_rest>false</is_rest>
+        <message/>
+        <name>Portfolio - Last updated changes</name>
+        <order>100</order>
+        <priority>100</priority>
+        <rest_method/>
+        <rest_method_text/>
+        <rest_service/>
+        <rest_service_text/>
+        <rest_variables/>
+        <role_conditions/>
+        <script><![CDATA[(function executeRule(current, previous /*null when async*/) {
+
+	var now = new GlideDateTime();
+	current.last_updated = now.getDisplayValueInternal();
+	
+	var message = "Portfolio " + current.name + " was updated.";
```

Comment:
Is this intentional? Just logging that a change happened?

Reply:
Yeah, to help show some kind of indicator when a BR runs, the others have done something similar so was keeping with it. It can be removed though. The `debug` is to show in the logs since `addInfoMessage` is only in the UI.